### DISCUSSION TITLE
Change The color palette?

### DISCUSSION THREAD

**Discussion by AlifCsEdu** [jeremyckahn/chitchatter#120] [Q&A]
I want to change the color palette from the default color palette that this project have. How and where do i do that?

such as this area background color : ![image](https://user-images.githubusercontent.com/96769230/228927756-dda2fe38-3f8b-4cf3-8a02-25a77385818a.png)

**Comment by jeremyckahn** [comment]
Hi @AlifCsEdu, Chitchatter doesn't offer custom theming in the app. However, you can fork and self-host a modified version of the app. If you go that route, you can customize the Material UI theme here: https://github.com/jeremyckahn/chitchatter/blob/94ed09c90fdb5ec6007c5b615de59e6b3ba0099a/src/components/Shell/Shell.tsx#L46-L50

You can define any theme property according to [Material UI's theme specification](https://mui.com/material-ui/customization/default-theme/).

**Reply by AlifCsEdu** [reply]
Okay, thanks finally! thanks 😃
